src/main/java/com/zaxxer/hikari/pool/ProxyConnection.java

```diff
@@ -455,11 +455,14 @@ private static Connection getClosedConnection()
             public Object invoke(Object proxy, Method method, Object[] args) throws Throwable
             {
                final String methodName = method.getName();
-               if ("abort".equals(methodName)) { 
+               if ("isValid".equals(methodName)) {
+                  return Boolean.FALSE;
+               }
+               else if ("close".equals(methodName)) { 
```

Comment:
Actually, I stand corrected.  The only way to get this delegate is to call ``unwrap()`` *after* already closing the connection.

We intercept ``isValid()`` because it is supposed to return ``false`` on closed connections.